### DISCUSSION TITLE
Unable to delete assignments

### DISCUSSION THREAD

**Discussion by Ujoa** [community/community#138122] [GitHub Education]
### Hi everyone,

My current classroom has duplicated assignments (not sure how that happened). As a result, I cannot delete either of them and I am also having issue reusing assignments across my different classrooms.

<img width="1037" alt="image" src="https://github.com/user-attachments/assets/66d8d0ae-938b-4f0d-9e85-04391d3e0e6e">

**Comment by Shriharsh-Deshmukh** [comment]
To resolve the duplicated assignments issue in GitHub Classroom:

1. **Manually Delete Duplicates:** If GitHub Classroom UI doesn’t allow deletion, try using the GitHub API to remove duplicate assignments directly from the repository.

2. **Reset Assignment States:** Go to the repository settings and try resetting or archiving the duplicated assignments manually.

3. **Resolve Reuse Issues:** Ensure that the assignment template repository permissions are correctly set (public or private), and check that the assignment is correctly linked across classrooms.

4. **Clear Cache/Incognito Mode:** If UI-related, clear your browser cache or try using incognito mode to check if the issue persists.

5. **GitHub Support:** If the issue remains unresolved, reach out to GitHub Support with detailed logs and screenshots.

This approach should fully resolve your issue.

**Comment by Injamul2k24** [comment]
Check for manual duplicates: The user should manually verify if the assignments are truly duplicates by comparing their content, due dates, and other details. If there are slight differences, they might be able to delete one of them.

Try creating a new assignment: The user could try creating a new assignment with the same content and settings as the desired assignment. If the platform allows, they might be able to reuse this new assignment across different classrooms.

Look for platform-specific solutions: The user should search for specific solutions related to their platform. There might be known workarounds or troubleshooting steps that can help resolve the issue.
